images/dev.Dockerfile

```diff
@@ -0,0 +1,39 @@
+#FROM mcr.microsoft.com/devcontainers/go:0-1-bullseye
+FROM almalinux:8
+
+# Create the xrootd user with a fixed GID/UID
+RUN groupadd -o -g 10940 xrootd
+RUN useradd -o -u 10940 -g 10940 -s /bin/sh xrootd
+
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+# Get goreleaser
+SHELL ["/bin/bash", "-c"]
+RUN echo $'[goreleaser] \n\
+name=GoReleaser \n\
+baseurl=https://repo.goreleaser.com/yum/ \n\
+enabled=1 \n\
+gpgcheck=0' > /etc/yum.repos.d/goreleaser.repo
+    
+# Install goreleaser and various other packages we need
+RUN yum install -y goreleaser npm xrootd xrootd-server xrootd-client nano xrootd-scitokens \
+    xrootd-voms jq procps docker make
+RUN yum clean all
+
+# Installing the right version of go
+SHELL ["/bin/sh", "-c"]
+RUN curl https://dl.google.com/go/go1.20.8.linux-arm64.tar.gz -o go1.20.8.linux-arm64.tar.gz 
```

Comment:
As it is, this only builds a container for arm64 -- great for us Mac users, but not useful for anyone else. Can we make two dev containers, one for amd64 and one for arm64? The amd64 container should use the OSG base containers. I couldn't use those for arm because the OSG repositories don't package xrootd for arm.

In the amd64 container, this line will need to be modified to grab the correct arch